tests/protocols/test_http.py

```diff
@@ -527,6 +527,28 @@ def test_root_path(protocol_cls):
     assert b"Path: /app/" in protocol.transport.buffer
 
 
+@pytest.mark.parametrize("protocol_cls", HTTP_PROTOCOLS)
+def test_raw_path(protocol_cls):
+    async def app(scope, receive, send):
+        path = scope["path"]
+        raw_path = scope.get("raw_path", None)
+        response = Response(
+            "Path: {}\nraw_path: {}".format(path, repr(raw_path)),
```

Comment:
Is the `repr` needed here?
Could we pull this out into two lines maybe:

```python
content = "path: {}\nraw_path: {}".format(path, raw_path)
response = Response(content, media_type="text/plain")
```

Reply:
I used the repr as a cheap-and-cheerful way of "proving" that the value was bytes, not a string while still being able to run assertions against the unicode string content for the response.

How about I change this to be nested assertions instead?

Reply:
Sure thing

Reply:
Done: https://github.com/encode/uvicorn/pull/372/commits/1c88abc91091a24fdc427ea8a9e973684f4c8e63